src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets

```diff
@@ -975,6 +980,26 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   </Target>
 
+  <!--
+    ============================================================
+                                        ComputeFilesCopiedToPublishDir
+
+    Gathers all the files that were copied to the publish directory.
+    ============================================================
+    -->
+  <Target Name="ComputeFilesCopiedToPublishDir"
+          DependsOnTargets="_CopyResolvedFilesToPublishPreserveNewest;
+                            _CopyResolvedFilesToPublishAlways;
+                            BundlePublishDirectory">
+
+    <ItemGroup>
+      <FilesCopiedToPublishDir Include="@(_ResolvedUnbundledFileToPublishPreserveNewest)"/>
```

Comment:
I'd recommend computing `FilesCopiedToPublishDir` within `PublishItemsOutputGroup`

Reply:
I intentionally am putting it in a separate target so that the desktop bridge (wapproj) targets can consume it without using PublishItemsOutputGroup.